modules/statistics/php/stats_demographic.class.inc

```diff
@@ -230,18 +238,19 @@ class Stats_Demographic extends \NDB_Form
         //REGISTERED CANDIDATES ROW
         $result = $DB->pselect(
             "SELECT s.subprojectid as rowid,
-                          count(DISTINCT(c.PSCID)) as val
-                      FROM candidate as c
-                          LEFT JOIN session s ON (s.CandID=c.CandID)
-                      WHERE c.CenterID <> '1'
-                          AND c.Entity_type != 'Scanner'
-                          AND c.Active = 'Y'
-                          $Param_Project
-                          $Param_Site
-                      GROUP BY s.subprojectid",
+                    count(DISTINCT(c.PSCID)) as val
+            FROM candidate as c
+            LEFT JOIN session s ON (s.CandID=c.CandID)
+            WHERE c.CenterID <> '1'
+            AND c.Entity_type != 'Scanner'
+            AND c.Active = 'Y'
+            $Param_Project
```

Comment:
These are hard-coded strings above so should be safe from SQLi. 👍 A comment would be nice just to reassure anyone who reviews this code in the future.